### ISSUE TITLE
Question - any special instructions for Active Directory

### ISSUE THREAD

**Issue by amitkoth** [XavRsl/Ldap#7]
I know this deals with LDAP search, but is there anything special to bear in mind when the directory is Active Directory, either in terms of config or searching?

**Comment by XavRsl** [comment]
Hi,
To be honest I don't have a clue... !!
I've made this package for my own needs which do not include AD. My best answer would be : have a go and tell me if it works. 
Sorry if my answer is not really helpful,

Xavier

**Comment by brunowego** [comment]
This works on OpenLDAP right @XavRsl ?

**Comment by XavRsl** [comment]
Yes. It would probably not work with active directory. If you feel like
making a fork, please do  !

Xavier
